client/src/router.ts

```diff
@@ -78,25 +72,33 @@ export const router = new Router({
     ],
 });
 
-router.beforeEach(async (to, from, next) => {
-    if (!coreStore.initialized && to.path !== "/_load") {
-        next({ path: "/_load" });
-        const response = await fetch("/api/auth");
-        if (response.ok) {
-            const data = await response.json();
-            if (data.auth) {
-                coreStore.setAuthenticated(true);
-                coreStore.setUsername(data.username);
-                coreStore.setEmail(data.email);
+router.beforeEach(async (to, _from, next) => {
```

Comment:
You never check the .authenticated field on auth records now, nor do you redirect people to /login.

I'm gonna keep reiterating this :p what is wrong with the old code and just changing the _load stuff to the new setLoading stuff ?

Reply:
Sorry, I (finally) get what you're saying now.

My latest commit has these changes.